mygame/app/repl.rb

```diff
@@ -0,0 +1,2 @@
+require_relative "emojis"
```

Comment:
repl.rb should probably be removed from this PR and gitignored since its contents change very quickly and depend on the dev I'd say ;)